WordPressAuthenticator/Navigation/NavigateToEnterWPCOMPassword.swift

```diff
@@ -0,0 +1,28 @@
+import Foundation
+
+/// Navigates to the WPCOM password flow.
+///
+public struct NavigateToEnterWPCOMPassword: NavigationCommand {
+    private let loginFields: LoginFields
+
+    public init(loginFields: LoginFields) {
+        self.loginFields = loginFields
+    }
+    public func execute(from: UIViewController?) {
+        let navigationController = (from as? UINavigationController) ?? from?.navigationController
+        presentPasswordView(navigationController: navigationController,
+                            loginFields: loginFields)
+    }
+}
+
+private extension NavigateToEnterWPCOMPassword {
+    func presentPasswordView(navigationController: UINavigationController?, loginFields: LoginFields) {
+        guard let controller = PasswordViewController.instantiate(from: .password) else {
+            WPAuthenticatorLogError("Failed to navigate to PasswordViewController from GetStartedViewController")
+            return
+        }
+
+        controller.loginFields = loginFields
```

Comment:
Same point as https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/784/files#r1350930454

Reply:
Similar to the above reply, we don't need to update those fields in this case.